geode-core/src/distributedTest/java/org/apache/geode/security/AuthExpirationDUnitTest.java

```diff
@@ -171,8 +170,7 @@ public void registeredInterest_slowReAuth_policyDefault() throws Exception {
   }
 
   @Test
-  @Ignore("unnecessary test case for re-auth, but it manifests GEODE-9704")
-  public void registeredInterest_slowReAuth_policyKeys_durableClient() throws Exception {
```

Comment:
We want this to be live test from now on.

Reply:
Based on the description of the JIRA:
>Run geode-core distributedTest AuthExpirationDUnitTest.registeredInterest_slowReAuth_policyKeys_durableClient(), change the InterestResultPolicy to NONE, you would see the test would fail occasionally

I think you have to change the InterestResultPolicy to NONE. So that it will test your fix.

Reply:
You are correct I think I lost that along the way.